certbot/cli.py

```diff
@@ -912,8 +913,9 @@ def prepare_and_parse_args(plugins, args, detect_defaults=False):  # pylint: dis
         help="Silence all output except errors. Useful for automation via cron."
              " Implies --non-interactive.")
     # overwrites server, handled in HelpfulArgumentParser.parse_args()
-    helpful.add("testing", "--test-cert", "--staging", action='store_true', dest='staging',
-        help='Use the staging server to obtain test (invalid) certs; equivalent'
+    helpful.add(["testing", "revoke"], "--test-cert", "--staging",
```

Comment:
It's a matter of judiciously selecting the most relevant options to be documented in those topics; `certonly` already has `--dry-run` which is probably better for most use cases, but I can add `--test-cert` to run.